doc/workshops/financial.rst

```diff
@@ -2,4 +2,1030 @@
+
+You can also type the name of the object (``netflix``)in the console, then type a period and hit the Tab key once. IPython suggestions will then appear to help you navigate the object:
+
+.. image:: images/workshop-1/ipython-auto-suggestions.png
+   :alt: IPython Console auto-suggestions
+
+For example, we can obtain general information with the ``info`` property of the object.
+
+.. code-block:: python
+
+   netflix_info = netflix.info
+
+We can observe the result in the Variable Explorer, in the ``netflix_info`` variable. 
+
+.. image:: images/workshop-1/net_flix-variable-in-variable-explorer.png
+   :alt: First variable in Variable Explorer
+
+If we double-click it, a new window with the detailed information will be displayed. 
+
+.. image:: images/workshop-1/dict-in-variable-explorer.png
+   :alt: Python dict example in Variable Explorer
+
+In that window we can see that a Python dictionary is displayed in which each key has a value assigned to it. Each type of value is represented by a distinctive color. For example, we see that Netflix is an Entertainment industry, the summary of the business type, among other indicators. The Variable Explorer is a very convenient way to view these types of results. We can compare it to viewing them in the Ipython Console with the ``pprint`` function:
+
+.. code-block:: python
+
+   pprint.pprint(netflix_info)
+
+Although ``pprint`` displays all the information, it is easier to view it in the Variable Explorer.
+
+
+Historical stock data
+~~~~~~~~~~~~~~~~~~~~~
+
+We can download in a dataset the history of a stock with the following line of code:
+
+.. code-block:: python
+
+   hist = netflix.history(period="max")
+
+We can see a summary and more details of this dataset in the Variable Explorer. It appears as a DataFrame object, with 7 columns and thousands of rows. If we double-click on it we will see the historical records of Netflix stock organized by ascending dates.
+
+.. image:: images/workshop-1/netflix-history.png
+   :alt: Netflix historical operations
+
+Throughout this workshop we will use this historical information to compare, with various normative financial theories, different types of financial portfolios.
+
+
+
+===============
+First portfolio
+===============
+
+Let's build our first stock investment portfolio! Say we are interested in investing in technology, and we want to know what performance can be obtained by putting money into some of the "heavyweights" in this industry.
+
+To measure the performance of our first portfolio we are going to use a classic theory in the world of finance: *mean-variance portfolio (MVP) theory*. This model assumes that investors only care about expected returns and the variance of such returns. The analysis is based entirely on statistical measures based on a time series of share prices, such as periodic mean returns and the variances of those returns with the same periodicity.
```

Comment:
It would be nice to have a link to some basic explanation of this theory (maybe Wikipedia)